yandex-speechkit/src/main/java/com/justai/aimybox/speechkit/yandex/cloud/YandexRecognitionApi.kt

```diff
@@ -17,7 +17,8 @@ internal class YandexRecognitionApi(
     private val yandexPassportOAuthKey: String,
```

Comment:
I'd get rid of this argument and replace it with `iAmTokenProvider` without the default value - let the user to define it explicitly.

Reply:
Old YandexRecognitionApi contructor calls will not compile. Is it ok?

Reply:
@kusraevs it's okay - we'll change the major version